src/System.Diagnostics.Debug/ref/System.Diagnostics.Debug.cs

```diff
@@ -8,105 +8,461 @@
 
 namespace System.Diagnostics
 {
+    /// <summary>
+    /// Provides a set of methods and properties that help debug your code. This class cannot be inherited.
```

Comment:
Ok, this is fixed in our docs now. It might be just easier to remove it from here too.